[Looker branch] Do not review; do not merge

julianhyde opened the pull request.

julianhyde committed:
Workaround

julianhyde committed:
Looker instructions

julianhyde committed:
Update Looker instructions, to allow publishing to Looker's Nexus using Gradle

The instructions documented here are those used to publish
calcite-looker-1.22.1-looker, the first looker release of
Calcite since Calcite switched from Maven to Gradle.

julianhyde committed:
Prepare for release 1.27.3-looker

julianhyde committed:
Temporary workaround: change SqlInternalOperator's syntax back to FUNCTION

danny0405 commented on looker.md:
```diff
@@ -0,0 +1,107 @@
+accidentally commit the secret credentials and expose them on a
+public site.
+
+Publish:
```
What, is this branch only for Looker ?

wnob committed:
[CALCITE-4872] Keep UNKNOWN distinct from NULL when switching nullability

wnob committed:
[CALCITE-4872] Fix bugs and incorporate feedback

wnob committed:
[CALCITE-4925] AggregateReduceFunctionsRule should accept arbitrary predicates

Allow selective application of the reduce-functions rule with arbitrary specificity.

wnob committed:
[CALCITE-4872] Incorporate feedback

julianhyde committed:
fixup 4872

julianhyde committed:
fixup 4925

julianhyde committed:
Prepare for release 1.27.4-looker
